src/app/course-page/course-page.component.ts

```diff
@@ -1,26 +1,31 @@
-import { Component, OnInit, Input } from '@angular/core';
+import { Component, OnInit, Input, ChangeDetectorRef } from '@angular/core';
 
-import { CourseService } from '../course.service';
+import { CourseService } from '../common/services/course.service';
 
-import { ICourse } from '../../common/course.interface';
-import { COURSES } from '../../common/constants/course-page.constants';
+import { ICourse } from '../common/course.interface';
+import { COURSES } from '../common/constants/course-page.constants';
+import { Router } from '@angular/router';
```

Comment:
move under `import { Component, OnInit, Input, ChangeDetectorRef } from '@angular/core';`

Reply:
To organize imports there are shortcuts in VSCode, e.g. Windows - Alt + Shift + O. VSCode help - [Organize imports](https://code.visualstudio.com/Docs/languages/typescript#_organize-imports)

Reply:
It only removes unused imports pack and sot by import path.
I prefer imports to be ordered like:
```
// Core imports

// Third-party imports

// Appication imports
/// Application Common imports
/// Local imports
```